API-LocateContractor/Controllers/TestController.cs

```diff
@@ -34,17 +34,9 @@ public ActionResult<List<Test>> Get()
 
             catch (Exception E)
             {
-
```

Comment:
Confused why your routing PR is making API changes

Reply:
In this case I was attempting to debug a minor issue. In the future I will be sure to push changes through a separate branch.

Reply:
All good, its not the end of the world. Just something to be cognizant of